crates/services/txpool/src/config.rs

```diff
@@ -63,7 +63,7 @@ impl Config {
             max_depth,
             min_gas_price,
             utxo_validation,
-            chain_config,
+            chain_parameters: chain_config,
```

Comment:
Maybe it is better to use one name, either `chain_config` or `chain_parameters` or `chain_params`. I prefer `chain_config` more=)